site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -262,6 +265,11 @@ Whether a parent email is required.
 [field]#{base_field_name}.familyConfiguration.parentRegistrationEmailTemplateId# [type]#[UUID]# [since]#Available since 1.8.0#::
 The unique Id of the email template to use for parent registration.
 
+[field]#{base_field_name}.formConfiguration.adminUserFormId# [type]#[UUID]# [since]#Available since 1.20.0#::
+The unique Id of the form to use for the Add and Edit User form when used in the FusionAuth admin UI. 
++
+A paid edition of FusionAuth is required to utilize custom forms. When this parameter is not provided it will default to a form provided by FusionAuth that may be used without a paid edition.
+
```

Comment:
Same comment as above, is this necessary, is it not formatting correctly without it?

Reply:
I mean, it's just an empty line in between the two. I can remove it though.

Reply:
Yes, remove it so we can keep each field definition the same. Did you see this done anywhere else?

Reply:
Throughout the file. But I will remove it.